server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionService.java

```diff
@@ -476,6 +475,11 @@ private Map<Long, CheckState> getTeamSpaceStatus(Set<Long> ids, Map<Long, Connec
         return connId2State;
     }
 
+    @SkipAuthorize("odc internal usage")
+    public ConnectionConfig nullSafeGet(@NonNull Long id) {
```

Comment:
why should we need this? where is the auth check?

Reply:
It is used for filtering unauthorized databases. For only internal usage, so without any auth check.

Reply:
I rename this method to `getBasicWithoutPermissionCheck` for better readability.